src/app/TargetSelect/TargetSelect.tsx

```diff
@@ -251,11 +263,24 @@ export const TargetSelect: React.FunctionComponent<TargetSelectProps> = (props)
 
   return (
     <>
-      <Card>
-        <CardHeader>
-          <CardHeaderMain>
+      <Card isExpanded={props.simple || selected === NO_TARGET ? undefined : isExpanded}>
+        <CardHeader
+          onExpand={props.simple || selected === NO_TARGET ? undefined : onExpand}
+          isToggleRightAligned={props.simple || selected === NO_TARGET ? undefined : true}
+          toggleButtonProps={
+            props.simple || selected === NO_TARGET
+              ? undefined
+              : {
+                  id: 'toggle-button1',
+                  'aria-label': 'Details',
+                  'aria-labelledby': 'expandable-card-title toggle-button1',
+                  'aria-expanded': isExpanded,
+                }
+          }
+        >
+          <CardTitle>
             <Text component={TextVariants.h4}>Target JVM</Text>
```

Comment:
Not sure why `h4` style is not being applied here. But using this fixes the issue:

https://www.patternfly.org/v4/components/card/#with-html-heading-element

```typescript
<CardTitle component="h4">
     Target JVM
</CardTitle>
```

Reply:
I'm not sure why any of our card titles were done the way they were at this point, either. `CardTitle` was just not being used, everything was `<CardHeaderMain><Text>foo</Text></CardHeaderMain>`. I've replaced those all with `CardTitle` so it's simpler and more idiomatic. This does mean that the card titles everywhere are now bolder, but that's just Patternfly default styling that we're now conformant to, so it's probably a good thing.

Reply:
Nice thanks :D